apps/web/pages/events.tsx

```diff
@@ -27,3 +38,21 @@ Events.Layout = HomeLayout;
 Events.RequireAuth = false;
 
 export default Events;
+
+export function SkeletonDemo() {
```

Comment:
- move file to comonents/preloader/componets   file named EventsLoader.tsx
- convert arrow function
- hard reset and remove pnpm file